src/server.ts

```diff
@@ -19,6 +19,7 @@ export default class CeramicAnchorServer extends Server {
   constructor(private container: DependencyContainer) {
     super(true);
 
+    this.app.set('trust proxy', true);
```

Comment:
wow, magic!  I'm just going to trust you that this is the proper magic 😅

Reply:
> I assume there needs to be a follow-up to make this actually show up as a field in the event logs?

No we are already capturing the proper field `ip`. This change sets that value to the X-Forwarded-For ip instead of the load balancer's ip.